webapp/channels/src/actions/websocket_actions.jsx

```diff
@@ -239,6 +241,7 @@ export function reconnect() {
         }
 
         dispatch(fetchAllMyTeamsChannels());
+        dispatch(fetchTeamScheduledPosts(currentTeamId, true));
```

Comment:
this is the actual change in order to get the scheduled messages slice refreshed on ws reconnect. The other part is for adding the time to the logged ws events information.